test/utils/domain/auctions/urls_test.js

```diff
@@ -0,0 +1,18 @@
+const rewire = require('rewire')
+const urls = rewire('../../../../utils/domain/auctions/urls')
+
+describe('Url Utilities', () => {
+  before(() => {
+    urls.__set__('PREDICTION_URL', 'http://www.liveauctions.com')
+  })
+  describe('liveAuctionUrl()', () => {
```

Comment:
Throughout Force people tend to write `describe('#liveAuctionUrl', () => { ... ` to denote functions (hash)